samtranslator/model/__init__.py

```diff
@@ -43,7 +43,8 @@ class Resource(object):
     property_types = None
     _keywords = ["logical_id", "relative_id", "depends_on", "resource_attributes"]
 
-    _supported_resource_attributes = ["DeletionPolicy", "UpdatePolicy", "Condition"]
+    _supported_resource_attributes = ["DeletionPolicy", "UpdatePolicy", "Condition", "UpdateReplacePolicy", "Metadata"]
+    _pass_through_attributes = ["Condition", "DeletionPolicy", "UpdateReplacePolicy"]
```

Comment:
Adding some documentation would be good for explaining which array works for what purpose.